File: app/src/main/java/uk/co/ourfriendirony/medianotifier/activities/ActivitySplash.java
package uk.co.ourfriendirony.medianotifier.activities;

import android.content.Intent;
import android.os.Bundle;
import androidx.appcompat.app.AppCompatActivity;

public class ActivitySplash extends AppCompatActivity {
    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);

        startActivity(new Intent(ActivitySplash.this, ActivityMain.class));
        finish();
    }
}

File: app/src/main/kotlin/uk/co/ourfriendirony/medianotifier/activities/ActivitySplash.kt
package uk.co.ourfriendirony.medianotifier.activities

import android.content.Intent
import android.os.Bundle
import androidx.appcompat.app.AppCompatActivity

class ActivitySplash : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        startActivity(Intent(this@ActivitySplash, ActivityMain::class.java))
        finish()
    }
}